### PR TITLE
Github action to automate publish of package when release is cut

### PR DESCRIPTION
**Pull request by shashank-boyapally**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Added automation to publish to pypi when release is cut. added setup.py for building of package. Request to the owner to add PYPI_API_TOKEN to the secrets of the repository. A smaller feature add is metadata in testing now supports fips, ipsec and encrypted.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: fmatch/matcher.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def burner_cpu_results(self, uuids, namespace, index):
                 "uuid": {
                     "terms": {
                         "field": "uuid.keyword",
-                        "size": self.searchSize
+                        "size": self.search_size
```
Comment (jtaleric) [inline fmatch/matcher.py:225]: ah, snake case vs camel case, my bad!

Comment (shashank-boyapally) [reply fmatch/matcher.py:225]: No problemo!! was getting an error in pylint, hence had to change.

**Comment by shashank-boyapally** [comment]
We need to change the version in setup.py everytime we cut a new release, as pypi does not allow to have the same version even though the package is deleted.

**Review by jtaleric — APPROVED** [review]

**Comment by shashank-boyapally** [comment]
Merging!!

**Pull request merged**